src/module.c

```diff
@@ -2254,6 +2254,34 @@ int moduleIsModuleCommand(void *module_handle, struct serverCommand *cmd) {
     return (cp->module == module_handle);
 }
 
+/* ValkeyModule_UpdateRuntimeArgs can be used to update the module argument values.
+ * The function parameter 'argc' indicates the number of updated arguments, and 'argv'
+ * represents the values of the updated arguments.
+ * Once 'CONFIG REWRITE' command is called, the updated argument values can be saved into conf file.
+ * One example can be found in file tests/modules/moduleparameter.c.
```

Comment:
We don't normally reference the test modules since these are rendered here: https://valkey.io/topics/modules-api-ref/, which doesn't clearly map to the test files anymore. I'm not sure it's really needed to show an example.

Reply:
It looks like no specific test file is mentioned in reference, I will remove it.